docs/setup.rst

```diff
@@ -724,6 +724,8 @@ configuring some options in paperless can help improve performance immensely:
     times. Thumbnails will be about 20% larger.
 *   If using docker, consider setting ``PAPERLESS_WEBSERVER_WORKERS`` to
     1. This will save some memory.
+*   Use the arm compatible docker-compose if you're wanting to use Tika on something like
+		a raspberry pi. The official apache/tika image does not support the arm architecture.
```

Comment:
Maybe there should also be the note, that the endpoints needs adjustment (see #363):
-> http://gotenberg:3000/forms/libreoffice/convert#

Reply:
Surely that doesn't belong in the _Considerations for less powerful devices_ section?

Reply:
#363 won't be considered solved yet, we still need to adjust the install script.